db/seeds.rb

```diff
@@ -6,31 +6,68 @@
-Language.find_or_create_by({id:1,name:'English',code:'EN'})
-Language.find_or_create_by({id:2,name:'Español',code:'ES'})
-Language.find_or_create_by({id:3,name:'Français',code:'FR'})
+
+Language.find_or_create_by(id: 1, name: 'English', code: 'EN')
+Language.find_or_create_by(id: 2, name: 'Español', code: 'ES')
+Language.find_or_create_by(id: 3, name: 'Français', code: 'FR')
 Language.connection.execute("select setval('languages_id_seq', (select max(id) from languages))")
 
 
-# note, these hardcoded id's are needed to match the migrated data
-Currency.find_or_create_by({id:1,name:'Argentinean Peso',code:'ARS',symbol:'AR$'})
-Currency.find_or_create_by({id:2,name:'U.S. Dollar',code:'USD',symbol:'US$'})
-Currency.find_or_create_by({id:3,name:'British Pound',code:'GBP',symbol:'GB£'})
-Currency.find_or_create_by({id:4,name:'Nicaraguan Cordoba',code:'NIO',symbol:'NI$'})
+Currency.find_or_create_by(id: 1, name: 'Argentinean Peso', code: 'ARS', symbol: 'AR$')
+Currency.find_or_create_by(id: 2, name: 'U.S. Dollar', code: 'USD', symbol: 'US$')
+Currency.find_or_create_by(id: 3, name: 'British Pound', code: 'GBP', symbol: 'GB£')
+Currency.find_or_create_by(id: 4, name: 'Nicaraguan Cordoba', code: 'NIO', symbol: 'NI$')
 Currency.connection.execute("select setval('currencies_id_seq', (select max(id) from currencies))")
 
 
-# note, these hardcoded id's are needed to match the migrated data
-Country.find_or_create_by({id:1,name:'Argentina',iso_code:'AR',default_language_id:2,default_currency_id:1})
-Country.find_or_create_by({id:2,name:'Nicaragua',iso_code:'NI',default_language_id:2,default_currency_id:4})
-Country.find_or_create_by({id:3,name:'United States',iso_code:'US',default_language_id:1,default_currency_id:2})
+Country.find_or_create_by(id: 1, name: 'Argentina', iso_code: 'AR', default_language_id: 2, default_currency_id: 1)
+Country.find_or_create_by(id: 2, name: 'Nicaragua', iso_code: 'NI', default_language_id: 2, default_currency_id: 4)
+Country.find_or_create_by(id: 3, name: 'United States', iso_code: 'US', default_language_id: 1, default_currency_id: 2)
 Country.connection.execute("select setval('countries_id_seq', (select max(id) from countries))")
 
+
 # for now mapping the '0' Person refs to 'null' and allowing null refs in the schema
 # Person.find_or_create_by({id:0, name: 'dummy', first_name: 'dummy', division_id: Division.root_id})
+
+# handy temporary change for debugging
+# Option.destroy_all
+# OptionSet.destroy_all
+
+loan_status = OptionSet.find_or_create_by(division: Division.root, model_type: Loan.name, model_attribute: 'status')
+# beware rerunning these different position values, will result in duplicated data
+# a more verbose approach would be needed if that is important to support
+# also we can consider dropping the 'position' assignments until an ordering different than 'value' is needed
+loan_status.options.find_or_create_by(value: 1, position: 1).set_label_list(en: 'Active', es: 'Prestamo Activo')
```

Comment:
I don't like the numeric values either.  It just happened to be the simplest approach given the need to migrate data.  Here is my suggestion of how to change:
1) make `value` a string value instead of an integer and use meaningful slugs for any special system configuration data populated by seeds.rb
2) in the future when we have user created custom option values, then we'll just automatically use the ID as the value
3) we add a `migration_id` field used for resolving the correct slug value when performing the data migration

This adds a little bit of complexity to the data migration, but I think that it's worthwhile in the interest of a better designed system in the long run.  I believe that there is legitimate business logic to be attached to the "active" and "completed" statuses.  But if that logic is now tied to meaningful slugs instead of arbitrary ids, then it feels clean to me.